app/code/Magento/Inventory/Model/ResourceModel/Reservation/ReservationQuantity.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\ResourceModel\Reservation;
```

Comment:
let's add `declare(strict_types=1);` for all newly created files in the scope of MSI project

Reply:
on which line do you prefer to add this directive?

* just after `<?php`, before copyrignt disclaimer comment
* just after copyrignt disclaimer comment
* just after`namespace` directive
* just after`namespace` and `use` directives

Reply:
just find this tool: https://github.com/dypa/declare-strict-types
didn't use it but maybe can be useful